crates/re_renderer/src/draw_phases/picking_layer.rs

```diff
@@ -26,17 +33,58 @@ pub struct PickingResult<T: 'static + Send + Sync> {
     /// Describes the area of the picking layer that was read back.
     pub rect: IntRect,
 
-    /// Picking data for the requested rectangle.
+    /// Picking id data for the requested rectangle.
+    ///
+    /// GPU internal row padding has already been removed.
+    /// Data is stored row wise, left to right, top to bottom.
```

Comment:
?

Reply:
Trying to clarify, but I'm just trying to say that the pixels in there are sane 😄

Reply:
No but I mean it's not like it's raw data?

Reply:
right, it's already converted 💡 
Changed it to "pixel data"